app/javascript/dashboard/components/SnackbarContainer.vue

```diff
@@ -42,7 +40,7 @@ export default {
     this.$emitter.off('newToastMessage', this.onNewToastMessage);
   },
   methods: {
-    onNewToastMessage(message, action) {
+    onNewToastMessage({ message, action }) {
```

Comment:
Why the need to convert this to an object?

Reply:
Oh just noticed the change in `composables/index.js` Any reason to do that?

Reply:
Fixes a bug from mitt migration

Reply:
@scmmishra The reason is that there is a component where we pass the action to the `useAlert`. So when I pass it as two params, the action does not show up, so I do it as an object. https://github.com/chatwoot/chatwoot/pull/9793/files#diff-e9818a3dc710645a0b30499e346262312210d60d68b54679e252769b012007b1R518

It was a bug that showed up after moving from bus to mitt.